test/user.js

```diff
@@ -0,0 +1,125 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  User = require('../app/models').users,
+  should = chai.should();
+
+beforeEach(async function() {
+  await User.deleteAll();
```

Comment:
You don't need this, check [previous](https://github.com/wolox-training/jl-express-js/pull/5/files#r221652387) comments